### ISSUE TITLE
The UI scroll does not show table info proportional to scroll position.

### ISSUE THREAD

**Issue by snenkov** [questdb/questdb#1020]
**Describe the bug**
If I move the side scroll of a table in the UI to the middle - I don't see the middle of the table. Instead I see different part of the table depending if I started from the bottom or the top of the table. 

**To Reproduce**
1. Use the demo table https://demo.questdb.io
2. Start from the top of the UI table and scroll down, once you get to the middle you will notice that the date is closer to thee start date (albeit older) When you get close to the bottom you don't get close to the end date. Once you get to the bottom you see the end date. Then you do the reverse form the bottom and the same happens (but in reverse); once you get the the middle you don't see the date before, but a a date closer to the bottom date.
In short -depending on where you start - the scroll does not move proportianlly

**Expected behavior**
I expect the scroll to be proportional - if I move in the middle - to see the middle table values, etc.

**Screenshots**
<img width="1661" alt="Screen Shot 2021-05-14 at 4 49 14 PM" src="https://user-images.githubusercontent.com/13110986/118288300-91f8cc00-b4d4-11eb-909d-34825a9663e5.png">
<img width="1640" alt="Screen Shot 2021-05-14 at 4 48 25 PM" src="https://user-images.githubusercontent.com/13110986/118288306-945b2600-b4d4-11eb-871d-c61e3a2bd4e2.png">

**Environment (please complete the following information):**
N/A since this is demo.

**Additional context**
N/A
